po/tr_TR.UTF-8.po

```diff
@@ -0,0 +1,270 @@
+# Turkish translations for com.mitchellh.ghostty package.
+# Copyright (C) 2025 Mitchell Hashimoto
+# This file is distributed under the same license as the com.mitchellh.ghostty package.
+# Emir SARI <emir_sari@icloud.com>, 2025.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.mitchellh.ghostty\n"
+"Report-Msgid-Bugs-To: m@mitchellh.com\n"
+"POT-Creation-Date: 2025-03-19 08:54-0700\n"
+"PO-Revision-Date: 2025-03-24 22:01+0300\n"
+"Last-Translator: Emir SARI <emir_sari@icloud.com>\n"
+"Language-Team: Turkish\n"
+"Language: tr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: src/apprt/gtk/ui/1.5/prompt-title-dialog.blp:5
+msgid "Change Terminal Title"
+msgstr "Uçbirim Başlığını Değiştir"
+
+#: src/apprt/gtk/ui/1.5/prompt-title-dialog.blp:6
+msgid "Leave blank to restore the default title."
+msgstr "Saptanmış başlığı geri yüklemek için boş bırakın."
```

Comment:
```suggestion
msgstr "Varsayılan başlığı geri yüklemek için boş bırakın."
```

Reply:
This is an interesting situation. We strive to be as native as possible in Ghostty and that includes using the preferred terminology for each platform. Would it make sense to translate these strings separately for Linux and macOS in this instance? (These are all currently Linux strings since macOS's localization support is still WIP)

Reply:
I think this is good to note and to consider later, but for now we can use the idiomatic Linux wording because these are Linux-only. There will be some cross-platform strings (i.e. config validation) but we aren't there yet. GUI text will probably always be platform-specific.

Reply:
Yes. macOS UI strings are highly polished and it is only natural that the software follow Apple's guidelines as much as possible.

https://www.applelocalization.com is a great source for checking out how the terminology is laid out.

Reply:
Also macOS uses typographic quotes and apostrophes, plus proper dashes wherever needed. It is also something to keep in mind while setting the strings.

Reply:
Those are recommended practices in Linux as well. We just haven't had any strings that necessitate using them yet :)

Reply:
Fixed.